### PR TITLE
Devkit is dead. Long live devkit.

### PR DESCRIPTION
**Pull request by mckern**
Devkit has had very little love over the years and the functionality
has been worked into tools like the `build`, `render`, and `inspect`
commands. Maybe it's time to let devkit rest.

----

The README probably needs an example of how to iterate on components
quickly -- maybe built around preserving remote workdirs (PR #55),
specifying the name of remote workdirs (PR #348, #452), and only
building up to specific components (PR #409).

----

No rush on this; it's something I sort of stumbled over when I started working on refactoring and updating the Docker engine. 

P.S. I'm refactoring and updating the Docker engine.

### CODE REVIEW

**Comment by mckern** [comment]
WTF Rubocop.

**Comment by McdonaldSeanp** [comment]
👍

**Comment by mckern** [comment]
Things are going to be broken until https://github.com/puppetlabs/lock_manager/pull/15 is merged. Hooray for Friday releases...

File: lib/vanagon/driver.rb [hunk 1]
```diff
@@ -143,36 +143,17 @@ def run # rubocop:disable Metrics/AbcSize
       end
     end
 
-    def render
+    def render # rubocop:disable Metrics/AbcSize
```
Comment (mwaggett) [inline lib/vanagon/driver.rb]: Is this necessary? How was this passing before?

Comment (mckern) [reply lib/vanagon/driver.rb]: I asked myself that same question about a dozen times, and I just have no idea why this stopped passing when I removed lines.

Comment (mwaggett) [reply lib/vanagon/driver.rb]: lol ¯\_(ツ)_/¯

**Comment by shrug** [comment]
lock_manager change is merged, we need to get a release pushed out

**Comment by shrug** [comment]
I think his is ok to merge now, are we still waiting on anything?

**Comment by haus** [comment]
Seems fine. My only nit would be that there are changes unrelated to removing devkit that could be in their own commit.

**Comment by mckern** [comment]
@haus I'll split it into two commits and yank the tags off this PR as soon as I get a chance

**Comment by shrug** [comment]
@mckern Are you still planning to split the commits?

**Comment by mckern** [comment]
140205_snut_7-10-split jpg crop promo-mediumlarge

**Comment by underscorgan** [comment]
Commits split!

**Comment by mckern** [comment]
mr meeseeks ooh yeah

whispers - ship it

**Comment by mckern** [comment]
(:heart: Morgan and TY for this)

**Pull request merged**